rspace/src/test/scala/coop/rchain/rspace/ReplayRSpaceTests.scala

```diff
@@ -57,20 +57,6 @@ trait ReplayRSpaceTests extends ReplayRSpaceTestsBase[String, Pattern, String, S
       space.store.isEmpty shouldBe true
   }
 
-  "Creating a COMM Event that is not contained in the trace log" should "throw a ReplayException" in
```

Comment:
This test should still pass.

Reply:
I agree. Why is it being removed?

Reply:
No it just doesn't reduce and thus at the end the state hashes will mismatch.

Reply:
Think about the case when there is a produce and consume that could reduce but they were originally part of different comm events.